static/js/components/EducationForm.js

```diff
@@ -25,23 +38,42 @@ import type {
   UpdateProfileFunc,
 } from '../flow/profileTypes';
 import type { UIState } from '../reducers/ui';
-import type { Validator, UIValidator } from '../lib/validation/profile';
+import type {
+  Validator,
+  UIValidator,
+} from '../lib/validation/profile';
+import ValidationAlert from './ValidationAlert';
 
 class EducationForm extends ProfileFormFields {
+  constructor(props: Object) {
+    super(props);
+
+    this.educationLevelLabels = {};
+    this.educationLevelOptions.forEach(level => {
+      this.educationLevelLabels[level.value] = level.label;
+    });
```

Comment:
what do you think about moving this out of the constructor and to either the `constants` file or to a module-level declaration in this file? there isn't any dependency on runtime data for this calculation.